app/Widgets/Literacy/README.md

```diff
@@ -0,0 +1,48 @@
+# Literacy for Digital Atlas
+
+This Literacy was designed for the Digital Atlas.  It ...
+
+## Dependencies
+
+It uses the following PHP libraries:
+
+- [arrilot/laravel-widgets](https://github.com/arrilot/laravel-widgets)
+
+## Install
+
+This package only works with the Digital Atlas.  To install:
+
+1. Drop this package into the *App/Widgets/* directory.
+2. Register the service provider, by adding the following to the providers array of *config/app.php* file:
+```
+App\Widgets\Literacy\LiteracyServiceProvider::class
+```
+3. In terminal, migrate the database.
+
+_Docker_
+```
+docker-compose run --rm da_artisan migrate
+```
+
+_Manual Installation_
+```
+php artisan migrate
+```
+
+4. Added the widget to the view file using `@asyncWidget('App\Widgets\Literacy\LiteracyWidget', [], $country)`.
+
+## Configuration
+
+To publish the configuration file, simply run the following command:
+
+_Docker_
+```
+docker-compose run --rm da_artisan vendor:publish
+```
+
```

Comment:
Can you add steps on how to import the data file?  If you need help, you can checkout the README file in the BroadbandSubscriptions widget.